src/test/java/com/microsoft/sqlserver/testframework/DBOptions.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * 
+ */
+package com.microsoft.sqlserver.testframework;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author v-afrafi
+ *
+ */
+public class DBOptions extends AbstractParentWrapper{
+    public static final Logger log = Logger.getLogger("DBOptions");
+
+    /**
+     * @param parent
+     * @param internal
+     * @param name
+     */
+    DBOptions(AbstractParentWrapper parent,
+            Object internal,
+            String name) {
+        super(parent, internal, name);
+        // TODO Auto-generated constructor stub
+    }
+
+    // 'SQL' represents SQL Server, while 'SQLAzure' represents SQL Azure. 
+    private static String _serverType         = null;  
+    public  static final  String  SERVER_TYPE_SQL_SERVER = "SQL";
+    public  static final  String  SERVER_TYPE_SQL_AZURE  = "SQLAzure";
+    
+    public static String servertype()
```

Comment:
- Formatting is not consistent for servertype method.
- Method name should be like getServerType(...) for better naming convention.
- Do you think this might be utility method and can be used by many test cases?